packages/react-native-reanimated/Common/cpp/reanimated/Fabric/EventLoop.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <reanimated/Tools/PlatformDepMethodsHolder.h>
+
+#include <functional>
+#include <memory>
+#include <unordered_map>
+#include <utility>
+
+namespace reanimated {
+
+class EventLoop {
```

Comment:
Since in JavaScript the `EventLoop` name is closely related to the Runtime, I'm not sure if it's the best name in this context 🤔. Could you also elaborate more on the purpose of this EventLoop? Feel free to explain it internally on Slack, if you prefer.